packages/api/endpoints/replays.js

```diff
@@ -43,6 +47,39 @@ async function startKinesisReplayAsyncOperation(req, res) {
   return res.status(202).send({ asyncOperationId: asyncOperation.id });
 }
 
+async function startSqsMessagesReplay(req, res) {
+  const tableName = process.env.AsyncOperationsTable;
+  const stackName = process.env.stackName;
+  const systemBucket = process.env.system_bucket;
+
+  const payload = req.body;
+  logger.debug(`Payload is ${JSON.stringify(payload)}`);
+
+  if (!payload.queueName) {
+    return res.boom.badRequest('queueName is required for SQS messages replay');
+  }
+
+  const queueUrl = await getQueueUrlByName(payload.queueName);
+  if (!(await sqsQueueExists(queueUrl))) {
+    logger.error(`Could not find queue ${queueUrl}. Unable to process message.`);
```

Comment:
```suggestion
    return res.boom.badRequest(`Could not find queue ${queueUrl}. Unable to process message.`);
```

We should just fail the request as bad at this point

Reply:
This is how I ended up changing it :[`fe6605f` (#2373)](https://github.com/nasa/cumulus/pull/2373/commits/fe6605f1129d0dfe2524419e4a4e2738c2d4abe8)
that point of the code was never reached in the case of a non existent queue since getQueueUrlByName calls `sqs().getQueueUrl()` which will actually do the throwing